### ISSUE TITLE
Ability to identify flakey / re-run tests

### ISSUE THREAD

**Issue by karan-jet** [runningcode/fladle#284]
Hi,

When using the re-run feature, I wanted to know if there is a way to identify tests that have been re-run / are flakey.
Is there an easy way to do this?

Many Thanks

**Comment by runningcode** [comment]
Yes, you can check the xml report. It will indicate which tests were flaky.
